Makefile

```diff
@@ -305,6 +304,37 @@ kernel: ## Build linux kernel.
 	docker cp $$id:/vmlinuz neonvm-kernel/vmlinuz; \
 	docker rm -f $$id
 
+.PHONY: kernel-source
+kernel-source: ## Set up git repo for the current kernel version and apply existing patches
+	kernel_version="$$(neonvm-kernel/echo-version.sh)"; \
```

Comment:
Consider extracting those make targets into helper scripts as well, would help with `; \`

Reply:
Discussed separately, planning to leave this for now - may revisit later & change to something easier to maintain.